test/check-application

```diff
@@ -28,6 +28,22 @@ IMG_BUSYBOX_LATEST = IMG_BUSYBOX + ":latest"
 IMG_REGISTRY = "localhost/test-registry"
 IMG_REGISTRY_LATEST = IMG_REGISTRY + ":latest"
 
+NGINX_DEFAULT_CONF = """
+server {
+  listen 80;
+  listen  [::]:80;
```

Comment:
Do we need  both?

Reply:
Can't say for sure but I think it's safer to keep both. `localhost` can resolve to ipv4 or ipv6

Reply:
Right, but `[::]` should listen on all interfaces already. Anyway, not important.